src/Moryx.AbstractionLayer.Resources.Endpoints/Converter/ResourceToModelConverter.cs

```diff
@@ -197,14 +197,8 @@ internal ResourceTypeModel ConvertType(IResourceTypeNode node, ResourceTypeModel
                 Creatable = node.Creatable,
                 Name = node.Name,
                 BaseType = baseType?.Name,
-
-                // Read display name of the type otherwise use type short name
-                DisplayName = resType.GetCustomAttribute<DisplayNameAttribute>(false)?.DisplayName ??
-                              Regex.Replace(resType.Name, @"`\d", string.Empty),
-
-                // Read description of the type
-                Description = resType.GetCustomAttribute<DescriptionAttribute>(false)?.Description,
-
+                DisplayName = resType.GetDisplayName(),
```

Comment:
What about the Fallback value based on TypeName?

Reply:
That is what the `ResourceTypeModel.Name` property is for, I don't think we should duplicate that information, when the UI want's to use the default value thats alright, but setting it in the endpoint seems not right for me 🤔

Reply:
Be that as it may, isn't it still a breaking change. If the UI used the display value reyling on the server it now shows empty

Reply:
Mhh true, but I don't like that we did this in the first place... 😑